RecoParticleFlow/PFClusterProducer/python/particleFlowZeroSuppressionECAL_cff.py

```diff
@@ -12,6 +12,49 @@
 _pfZeroSuppressionThresholds_EEplus_2017 = _pfZeroSuppressionThresholds_EEminus_2017
 
 
+# 
+# The three different set of thresholds will be used to study
+# possible new thresholds of pfrechits and effects on high level objects
+# The values proposed (A, B, C) are driven by expected noise levels
+# A ~ 0.5 sigma noise equivalent thresholds
+# B ~ 1.0 sigma noise equivalent thresholds
+# C ~ 2.0 sigma noise equivalent thresholds
```

Comment:
looking at the values, it appears that the description above is in a wrong order.
E.g. in EB A has 0.18, B has 0.14, and C has 0.10.

@amassiro please check and update if indeed incorrect, otherwise please clarify the math (why ```0.5*sigma``` is larger than ```2.0*sigma```